crates/astria-composer/env.example

```diff
@@ -12,3 +12,6 @@ ASTRIA_COMPOSER_CHAIN_ID="912559"
 
 # Address of the RPC server for execution
 ASTRIA_COMPOSER_EXECUTION_URL="ws://127.0.0.1:8545"
+
+# Private key for the sequencer account used for signing transactions
```

Comment:
As an afterthought: can you add that this is supposed to be a hex encoded 32 byte array (meaning that the key has to be 64 chars long)?